tests/fixtures/issues/issue-033.php

```diff
@@ -15,7 +15,7 @@
             'MinorVer' => '6',
             'Platform' => 'MacOSX',
             'Platform_Version' => '10.5',
-            'Platform_Description' => 'Mac OS X',
+            'Platform_Description' => 'Mac OS X for Power PC',
```

Comment:
With the addition of the `OSX_PPC` platform, there are a lot of tests with this change.